portal-impl/src/com/liferay/portal/service/impl/PortletLocalServiceImpl.java

```diff
@@ -2592,7 +2631,7 @@ protected Portlet updatePortlet(
 
 	private static final Map<String, PortletApp> _portletApps =
 		new ConcurrentHashMap<>();
-	private static final Map<String, String> _portletIdsByStrutsPath =
+	private static volatile Map<String, String> _portletIdsByStrutsPath =
 		new ConcurrentHashMap<>();
```

Comment:
Since we are changing this map to be copy on write, it should not have an initial value, right?